src/components/molecules/UserProfilePicture/UserProfilePicture.scss

```diff
@@ -1,83 +1,152 @@
+$reactionSize: calc(#{$avatarSize / 1.5});
+$reactionOffset: calc(-1 * #{$reactionSize} / 2);
+$shoutBounceTranslateXOffset: calc(#{$avatarSize} / 2);
+
+.UserProfilePicture {
+  width: 100%;
+  height: 100%;
+
   background-position: center;
   background-size: cover;
 
-  .profile-icon {
-    width: 44px;
-    height: 44px;
-    background-size: 100%;
+  &--reaction-left {
+    .UserProfilePicture__reaction {
+      left: $reactionOffset;
+    }
+
+    .UserProfilePicture__shout {
+      right: 0;
+      transform-origin: right center;
+      animation: shout-bounce--left 5s ease;
+    }
+  }
+
+  &--reaction-right {
+    .UserProfilePicture__reaction {
+      right: $reactionOffset;
+    }
+
+    .UserProfilePicture__shout {
+      left: 0;
+      transform-origin: calc(#{$avatarSize} / 2) center;
+      animation: shout-bounce--right 5s ease;
+    }
+  }
+
+  &__avatar {
+    width: $avatarSize;
+    height: $avatarSize;
+
+    background-color: transparent;
+    // TODO: how can we use a default image here if none set? Can we use CSS --variables here? I think we could..
+    //background-image: url(${({ backgroundImage }) => backgroundImage ?? DEFAULT_PROFILE_IMAGE});
+    background-position: center;
+    background-size: cover;
     background-repeat: no-repeat;
+
+    border-radius: 10rem;
+  }
+
+  &__reaction-container {
+    position: relative;
+
+    top: $reactionContainerOffset;
+    //z-index: -1;
+    //top: 0;
+    //left: 0;
+    width: $avatarSize;
+    height: $avatarSize;
+    //width: 100%;
+    //height: 100%;
+
+    // TODO: remove this
+    background-color: rgba(255, 0, 0, 0.2);
+    border: 1px solid #f00;
+  }
+
+  &__reaction {
+    position: absolute;
+    top: $reactionOffset;
+
+    // Note: emoji reactions are sized with font-size
+    font-size: $reactionSize;
+
+    animation: reaction-pulse 4s ease-in-out infinite;
   }
 
-  .messageToBand {
+  &__shout {
+    position: absolute;
+    bottom: 0;
+
     width: max-content;
     max-width: 20em;
     padding: 6px 10px;
 
-    position: absolute;
-    top: 0;
-    left: 0;
-
-    background: $fadedWhite;
+    // TODO: this is how the old styled-components had it
+    //background-color: rgba(255, 255, 255, 1);
+    // TODO: this is how the old (potentially unused) scss had it
+    background-color: $fadedWhite;
 
     color: $black;
     font-size: 20px;
     border-radius: 10px;
-
-    transform-origin: 2vh center;
-
-    animation: expand-bounce 5s ease;
   }
+}
 
-  .reaction {
-    width: 50px;
+@keyframes reaction-pulse {
+  0% {
+    transform: scale(1);
+  }
 
-    position: absolute;
-    top: -25px;
-    right: -20px;
-    z-index: z(user-profile-picture-reaction);
+  50% {
+    transform: scale(1.3);
+  }
 
-    font-size: 50px;
+  100% {
+    transform: scale(1);
+  }
+}
 
-    animation: pulse 4s ease-in-out infinite;
+// TODO: can we de-duplicate these with SCSS helper functions/etc?
+@keyframes shout-bounce--right {
+  0%,
+  100% {
```

Comment:
[TODO] can we de-duplicate these with SCSS helper functions/etc?

Currently the `--left` and `--right` variants are practically identical, it would be nice to not have to duplicate them.

Reply:
Calling this OOS